dp/cloud/python/magma/configuration_controller/config.py

```diff
@@ -53,6 +54,18 @@ class Config(object):
         'SAS_CERT_PATH', '/backend/configuration_controller/certs/ca.crt',
     )
 
+    # Elasticsearch
+    ELASTICSEARCH_INDEX = os.environ.get('ELASTICSEARCH_INDEX', 'dp')
```

Comment:
Can we have a separate module for elastic search?
It doesn't have to be a separate app (with Dockerfile), it can just be simple python module with:
- config (so that CC and RC don't have to have duplicated parts)
- one method publish request (or something like that, used by both RC and CC)